src/SmartEntity/FlexEntity.php

```diff
@@ -111,16 +111,18 @@ protected function addArrayField(&$array, $value, array $arguments)
     {
         if (isset($arguments[1])) {
             $array[(string) $arguments[1]] = $value;
-            return $array;
+
+        return $array;
```

Comment:
```suggestion
            return $array;
```
My bad, done it on the web platform, didn't see the right amount of spaces